Homework №3 from 21.09.2022/task1.c

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include <stdlib.h>
+#define lenOfTestArrs 9
+
+void printArr(int *array, int lenOfArray)
+{
+    for (int i = 0; i < lenOfArray; ++i)
+    {
+        printf("%d ", array[i]);
+    }
+    printf("\n");
+}
+
+void insertionSort(int *arr, int arrStart, int arrEnd)
+{
+    int separator = arrStart;
```

Comment:
```suggestion
    int currentElement = 0;
```
Почему переменная разделителем называется?
И зачем она инициализирована левым индексом, если далее в этой переменной не индексы, а элементы массива будут храниться?